packages/main/test/samples/TextArea.sample.html

```diff
@@ -26,4 +26,16 @@ <h3>TextArea with Maximum Length</h3>
 	</xmp></pre>
 </section>
 
+<section>
+	<h3>TextArea with Label</h3>
+	<div class="snippet">
+		<ui5-label for="taWithLabelID">Description</ui5-label>
```

Comment:
taWithLabelID
this sounds strange

Reply:
This is the sample page and the ID is used for the unit tests.